sdk/remoterendering/mixedreality-remoterendering/review/mixedreality-remoterendering.api.md

```diff
@@ -0,0 +1,276 @@
+
+```ts
+
+import { AccessToken } from '@azure/core-auth';
+import { AzureKeyCredential } from '@azure/core-auth';
+import { OperationOptions } from '@azure/core-client';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PipelineOptions } from '@azure/core-rest-pipeline';
+import { PollerLike } from '@azure/core-lro';
+import { PollOperationState } from '@azure/core-lro';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export type AssetConversion = NonStartedAssetConversion | RunningAssetConversion | SucceededAssetConversion | FailedAssetConversion | CancelledAssetConversion;
+
+// @public
+export interface AssetConversionBase {
+    conversionId: string;
+    createdOn: Date;
+    settings: AssetConversionSettings;
+}
+
+// @public
+export interface AssetConversionInputSettings {
+    blobPrefix?: string;
+    relativeInputAssetPath: string;
+    storageContainerReadListSas?: string;
+    storageContainerUrl: string;
+}
+
+// @public
+export interface AssetConversionOperationState extends PollOperationState<AssetConversion> {
+    latestResponse: AssetConversion;
+}
+
+// @public
+export interface AssetConversionOutput {
+    readonly outputAssetUrl?: string;
+}
+
+// @public
+export interface AssetConversionOutputSettings {
+    blobPrefix?: string;
+    outputAssetFilename?: string;
+    storageContainerUrl: string;
+    storageContainerWriteSas?: string;
+}
+
+// @public
+export type AssetConversionPollerLike = PollerLike<AssetConversionOperationState, AssetConversion>;
+
+// @public
+export interface AssetConversionPollerOptions {
+    intervalInMs?: number;
+}
+
+// @public
+export interface AssetConversionSettings {
+    inputSettings: AssetConversionInputSettings;
+    outputSettings: AssetConversionOutputSettings;
+}
+
+// @public
+export type AssetConversionStatus = string;
+
+// @public
+export type BeginConversionOptions = AssetConversionPollerOptions & OperationOptions;
+
+// @public
+export type BeginSessionOptions = RenderingSessionPollerOptions & OperationOptions;
+
+// @public
+export interface CancelledAssetConversion extends AssetConversionBase {
+    status: "Cancelled";
+}
+
+// @public
+export type EndSessionOptions = OperationOptions;
+
+// @public
+export interface ErrorRenderingSession extends RenderingSessionBase {
+    readonly error: RemoteRenderingServiceError;
+    partialProperties: PartialRenderingSessionProperties;
+    status: "Error";
+}
+
+// @public
+export interface ExpiredRenderingSession extends RenderingSessionBase {
+    properties: RenderingSessionProperties;
+    status: "Expired";
+}
+
+// @public
+export interface FailedAssetConversion extends AssetConversionBase {
+    error: RemoteRenderingServiceError;
+    status: "Failed";
+}
+
+// @public
+export type GetConversionOptions = OperationOptions;
+
+// @public
+export type GetSessionOptions = OperationOptions;
+
+// @public
+export enum KnownAssetConversionStatus {
```

Comment:
The "KnownAssetConversionStatus" enum is really a carrier for values, and isn't never used as a type. "AssetConversionStatus" is a type (defined to be string), and it's possible you intended to suggest that we change that. 

If this is the case, then it would make this a non-extensible enum. In fact, there is some benefit for extensibility here.

Reply:
This was me not getting we're not using this type directly but providing it as a reference.